client/src/components/Calendar/index.js

```diff
@@ -0,0 +1,70 @@
+import React, {Component} from 'react';
+import Calendar from "react-big-calendar";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+import moment from "moment";
+import axios from 'axios';
+
+class BigCalendar extends Component {
+
+  state={
+     loading: false
+   }
+  componentDidMount(){
+     axios.get('/events')
+     .then((data)=>{
+       const result =data.data;
+       const events = result.map((event)=>{
+        return { id: event.id,
+         title: event.title,
+         start: new Date(event.start_date),
+         end: new Date (event.end_date),
+         status: event.status,
+         price: event.price,
+         capacity: event.capacity,
+         note: event.note,
+         orgName: event.org_name,
+         userId : event.eventId}
+       })
+       this.setState({
+         events,
+         loading:true
+       })
+     }).catch((error)=>{
+
```

Comment:
You have to handle the error that you may receive it during fetching data

Reply:
done :+1: